# Copyright (C) 2001-2020, Python Software Foundation
# This file is distributed under the same license as the Python package.
# Maintained by the python-doc-es workteam. 
# docs-es@python.org / https://mail.python.org/mailman3/lists/docs-es.python.org/
# Check https://github.com/PyCampES/python-docs-es/blob/3.8/TRANSLATORS to get the list of volunteers
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Python 3.8\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-05-06 11:59-0400\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: python-doc-es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../Doc/library/errno.rst:2
msgid ":mod:`errno` --- Standard errno system symbols"
msgstr ""

#: ../Doc/library/errno.rst:9
msgid ""
"This module makes available standard ``errno`` system symbols. The value of "
"each symbol is the corresponding integer value. The names and descriptions "
"are borrowed from :file:`linux/include/errno.h`, which should be pretty all-"
"inclusive."
msgstr ""

#: ../Doc/library/errno.rst:17
msgid ""
"Dictionary providing a mapping from the errno value to the string name in "
"the underlying system.  For instance, ``errno.errorcode[errno.EPERM]`` maps "
"to ``'EPERM'``."
msgstr ""

#: ../Doc/library/errno.rst:21
msgid ""
"To translate a numeric error code to an error message, use :func:`os."
"strerror`."
msgstr ""

#: ../Doc/library/errno.rst:23
msgid ""
"Of the following list, symbols that are not used on the current platform are "
"not defined by the module.  The specific list of defined symbols is "
"available as ``errno.errorcode.keys()``.  Symbols available can include:"
msgstr ""

#: ../Doc/library/errno.rst:30
msgid "Operation not permitted"
msgstr ""

#: ../Doc/library/errno.rst:35
msgid "No such file or directory"
msgstr ""

#: ../Doc/library/errno.rst:40
msgid "No such process"
msgstr ""

#: ../Doc/library/errno.rst:45
msgid "Interrupted system call."
msgstr ""

#: ../Doc/library/errno.rst:48
msgid "This error is mapped to the exception :exc:`InterruptedError`."
msgstr ""

#: ../Doc/library/errno.rst:53
msgid "I/O error"
msgstr ""

#: ../Doc/library/errno.rst:58
msgid "No such device or address"
msgstr ""

#: ../Doc/library/errno.rst:63
msgid "Arg list too long"
msgstr ""

#: ../Doc/library/errno.rst:68
msgid "Exec format error"
msgstr ""

#: ../Doc/library/errno.rst:73
msgid "Bad file number"
msgstr ""

#: ../Doc/library/errno.rst:78
msgid "No child processes"
msgstr ""

#: ../Doc/library/errno.rst:83
msgid "Try again"
msgstr ""

#: ../Doc/library/errno.rst:88
msgid "Out of memory"
msgstr ""

#: ../Doc/library/errno.rst:93
msgid "Permission denied"
msgstr ""

#: ../Doc/library/errno.rst:98
msgid "Bad address"
msgstr ""

#: ../Doc/library/errno.rst:103
msgid "Block device required"
msgstr ""

#: ../Doc/library/errno.rst:108
msgid "Device or resource busy"
msgstr ""

#: ../Doc/library/errno.rst:113
msgid "File exists"
msgstr ""

#: ../Doc/library/errno.rst:118
msgid "Cross-device link"
msgstr ""

#: ../Doc/library/errno.rst:123
msgid "No such device"
msgstr ""

#: ../Doc/library/errno.rst:128
msgid "Not a directory"
msgstr ""

#: ../Doc/library/errno.rst:133
msgid "Is a directory"
msgstr ""

#: ../Doc/library/errno.rst:138
msgid "Invalid argument"
msgstr ""

#: ../Doc/library/errno.rst:143
msgid "File table overflow"
msgstr ""

#: ../Doc/library/errno.rst:148
msgid "Too many open files"
msgstr ""

#: ../Doc/library/errno.rst:153
msgid "Not a typewriter"
msgstr ""

#: ../Doc/library/errno.rst:158
msgid "Text file busy"
msgstr ""

#: ../Doc/library/errno.rst:163
msgid "File too large"
msgstr ""

#: ../Doc/library/errno.rst:168
msgid "No space left on device"
msgstr ""

#: ../Doc/library/errno.rst:173
msgid "Illegal seek"
msgstr ""

#: ../Doc/library/errno.rst:178
msgid "Read-only file system"
msgstr ""

#: ../Doc/library/errno.rst:183
msgid "Too many links"
msgstr ""

#: ../Doc/library/errno.rst:188
msgid "Broken pipe"
msgstr ""

#: ../Doc/library/errno.rst:193
msgid "Math argument out of domain of func"
msgstr ""

#: ../Doc/library/errno.rst:198
msgid "Math result not representable"
msgstr ""

#: ../Doc/library/errno.rst:203
msgid "Resource deadlock would occur"
msgstr ""

#: ../Doc/library/errno.rst:208
msgid "File name too long"
msgstr ""

#: ../Doc/library/errno.rst:213
msgid "No record locks available"
msgstr ""

#: ../Doc/library/errno.rst:218
msgid "Function not implemented"
msgstr ""

#: ../Doc/library/errno.rst:223
msgid "Directory not empty"
msgstr ""

#: ../Doc/library/errno.rst:228
msgid "Too many symbolic links encountered"
msgstr ""

#: ../Doc/library/errno.rst:233
msgid "Operation would block"
msgstr ""

#: ../Doc/library/errno.rst:238
msgid "No message of desired type"
msgstr ""

#: ../Doc/library/errno.rst:243
msgid "Identifier removed"
msgstr ""

#: ../Doc/library/errno.rst:248
msgid "Channel number out of range"
msgstr ""

#: ../Doc/library/errno.rst:253
msgid "Level 2 not synchronized"
msgstr ""

#: ../Doc/library/errno.rst:258
msgid "Level 3 halted"
msgstr ""

#: ../Doc/library/errno.rst:263
msgid "Level 3 reset"
msgstr ""

#: ../Doc/library/errno.rst:268
msgid "Link number out of range"
msgstr ""

#: ../Doc/library/errno.rst:273
msgid "Protocol driver not attached"
msgstr ""

#: ../Doc/library/errno.rst:278
msgid "No CSI structure available"
msgstr ""

#: ../Doc/library/errno.rst:283
msgid "Level 2 halted"
msgstr ""

#: ../Doc/library/errno.rst:288
msgid "Invalid exchange"
msgstr ""

#: ../Doc/library/errno.rst:293
msgid "Invalid request descriptor"
msgstr ""

#: ../Doc/library/errno.rst:298
msgid "Exchange full"
msgstr ""

#: ../Doc/library/errno.rst:303
msgid "No anode"
msgstr ""

#: ../Doc/library/errno.rst:308
msgid "Invalid request code"
msgstr ""

#: ../Doc/library/errno.rst:313
msgid "Invalid slot"
msgstr ""

#: ../Doc/library/errno.rst:318
msgid "File locking deadlock error"
msgstr ""

#: ../Doc/library/errno.rst:323
msgid "Bad font file format"
msgstr ""

#: ../Doc/library/errno.rst:328
msgid "Device not a stream"
msgstr ""

#: ../Doc/library/errno.rst:333
msgid "No data available"
msgstr ""

#: ../Doc/library/errno.rst:338
msgid "Timer expired"
msgstr ""

#: ../Doc/library/errno.rst:343
msgid "Out of streams resources"
msgstr ""

#: ../Doc/library/errno.rst:348
msgid "Machine is not on the network"
msgstr ""

#: ../Doc/library/errno.rst:353
msgid "Package not installed"
msgstr ""

#: ../Doc/library/errno.rst:358
msgid "Object is remote"
msgstr ""

#: ../Doc/library/errno.rst:363
msgid "Link has been severed"
msgstr ""

#: ../Doc/library/errno.rst:368
msgid "Advertise error"
msgstr ""

#: ../Doc/library/errno.rst:373
msgid "Srmount error"
msgstr ""

#: ../Doc/library/errno.rst:378
msgid "Communication error on send"
msgstr ""

#: ../Doc/library/errno.rst:383
msgid "Protocol error"
msgstr ""

#: ../Doc/library/errno.rst:388
msgid "Multihop attempted"
msgstr ""

#: ../Doc/library/errno.rst:393
msgid "RFS specific error"
msgstr ""

#: ../Doc/library/errno.rst:398
msgid "Not a data message"
msgstr ""

#: ../Doc/library/errno.rst:403
msgid "Value too large for defined data type"
msgstr ""

#: ../Doc/library/errno.rst:408
msgid "Name not unique on network"
msgstr ""

#: ../Doc/library/errno.rst:413
msgid "File descriptor in bad state"
msgstr ""

#: ../Doc/library/errno.rst:418
msgid "Remote address changed"
msgstr ""

#: ../Doc/library/errno.rst:423
msgid "Can not access a needed shared library"
msgstr ""

#: ../Doc/library/errno.rst:428
msgid "Accessing a corrupted shared library"
msgstr ""

#: ../Doc/library/errno.rst:433
msgid ".lib section in a.out corrupted"
msgstr ""

#: ../Doc/library/errno.rst:438
msgid "Attempting to link in too many shared libraries"
msgstr ""

#: ../Doc/library/errno.rst:443
msgid "Cannot exec a shared library directly"
msgstr ""

#: ../Doc/library/errno.rst:448
msgid "Illegal byte sequence"
msgstr ""

#: ../Doc/library/errno.rst:453
msgid "Interrupted system call should be restarted"
msgstr ""

#: ../Doc/library/errno.rst:458
msgid "Streams pipe error"
msgstr ""

#: ../Doc/library/errno.rst:463
msgid "Too many users"
msgstr ""

#: ../Doc/library/errno.rst:468
msgid "Socket operation on non-socket"
msgstr ""

#: ../Doc/library/errno.rst:473
msgid "Destination address required"
msgstr ""

#: ../Doc/library/errno.rst:478
msgid "Message too long"
msgstr ""

#: ../Doc/library/errno.rst:483
msgid "Protocol wrong type for socket"
msgstr ""

#: ../Doc/library/errno.rst:488
msgid "Protocol not available"
msgstr ""

#: ../Doc/library/errno.rst:493
msgid "Protocol not supported"
msgstr ""

#: ../Doc/library/errno.rst:498
msgid "Socket type not supported"
msgstr ""

#: ../Doc/library/errno.rst:503
msgid "Operation not supported on transport endpoint"
msgstr ""

#: ../Doc/library/errno.rst:508
msgid "Protocol family not supported"
msgstr ""

#: ../Doc/library/errno.rst:513
msgid "Address family not supported by protocol"
msgstr ""

#: ../Doc/library/errno.rst:518
msgid "Address already in use"
msgstr ""

#: ../Doc/library/errno.rst:523
msgid "Cannot assign requested address"
msgstr ""

#: ../Doc/library/errno.rst:528
msgid "Network is down"
msgstr ""

#: ../Doc/library/errno.rst:533
msgid "Network is unreachable"
msgstr ""

#: ../Doc/library/errno.rst:538
msgid "Network dropped connection because of reset"
msgstr ""

#: ../Doc/library/errno.rst:543
msgid "Software caused connection abort"
msgstr ""

#: ../Doc/library/errno.rst:548
msgid "Connection reset by peer"
msgstr ""

#: ../Doc/library/errno.rst:553
msgid "No buffer space available"
msgstr ""

#: ../Doc/library/errno.rst:558
msgid "Transport endpoint is already connected"
msgstr ""

#: ../Doc/library/errno.rst:563
msgid "Transport endpoint is not connected"
msgstr ""

#: ../Doc/library/errno.rst:568
msgid "Cannot send after transport endpoint shutdown"
msgstr ""

#: ../Doc/library/errno.rst:573
msgid "Too many references: cannot splice"
msgstr ""

#: ../Doc/library/errno.rst:578
msgid "Connection timed out"
msgstr ""

#: ../Doc/library/errno.rst:583
msgid "Connection refused"
msgstr ""

#: ../Doc/library/errno.rst:588
msgid "Host is down"
msgstr ""

#: ../Doc/library/errno.rst:593
msgid "No route to host"
msgstr ""

#: ../Doc/library/errno.rst:598
msgid "Operation already in progress"
msgstr ""

#: ../Doc/library/errno.rst:603
msgid "Operation now in progress"
msgstr ""

#: ../Doc/library/errno.rst:608
msgid "Stale NFS file handle"
msgstr ""

#: ../Doc/library/errno.rst:613
msgid "Structure needs cleaning"
msgstr ""

#: ../Doc/library/errno.rst:618
msgid "Not a XENIX named type file"
msgstr ""

#: ../Doc/library/errno.rst:623
msgid "No XENIX semaphores available"
msgstr ""

#: ../Doc/library/errno.rst:628
msgid "Is a named type file"
msgstr ""

#: ../Doc/library/errno.rst:633
msgid "Remote I/O error"
msgstr ""

#: ../Doc/library/errno.rst:638
msgid "Quota exceeded"
msgstr ""
